homeassistant/components/ecovacs/image.py

```diff
@@ -47,6 +50,7 @@ def __init__(
         """Initialize entity."""
         super().__init__(device, capability, hass=hass)
         self._attr_extra_state_attributes = {}
+        self._map = cast(Map, self._device.map)
```

Comment:
If the bot has a map will be checked above. See https://github.com/home-assistant/core/blob/edenhaus-deebot-client-13.0.0b0/homeassistant/components/ecovacs/image.py#L29